### ISSUE TITLE
Error code 500 (Cannot find server.): GET https://fc.yahoo.com

### ISSUE THREAD

**Issue by memoryfraction** [lppkarl/YahooFinanceApi#70]
Flurl.Http.FlurlHttpException: Call failed with status code 500 (Cannot find server.): GET https://fc.yahoo.com
![image](https://github.com/karlwancl/YahooFinanceApi/assets/18358791/e03b928a-55cc-4108-b8c0-69c854675c9d)

Code
```
[TestMethod]
public async Task GetDailyData_Should_Work()
{
    // 设置时间范围（过去1年）
    var toDate = DateTime.UtcNow;
    var fromDate = toDate.AddMonths(-1);

    // 创建一个标的（BTC/USD）并请求历史数据
    var symbol = "BTC-USD";
    var candles = await Yahoo.GetHistoricalAsync(symbol, fromDate, toDate, Period.Daily); // Daily, Weekly, Monthly

    // 访问OHLCV数据
    foreach (var candle in candles)
    {
        Console.WriteLine($"Date: {candle.DateTime}, Open: {candle.Open}, High: {candle.High}, Low: {candle.Low}, Close: {candle.Close}, Volume: {candle.Volume}");
    }
    Assert.IsTrue(candles != null && candles.Any());
}

```

**Comment by memoryfraction** [comment]
seems it's an error caused by yahoo finance from yesterday
But finance(a python package), is working at the same time. I don't know why
![image](https://github.com/karlwancl/YahooFinanceApi/assets/18358791/997edb31-82a5-4536-98cb-b0a8f4408423)
Comment: the printscreen is from： https://fc.yahoo.com/

**Comment by HanJaeJoon** [comment]
@memoryfraction 
Could you try to upgrade your nuget package version?
I had the same issue, but it was resolved after the upgrade.
This seems to be resolved in #66
